frontend/catalyst/third_party/oqc/src/CMakeLists.txt

```
@@ -10,18 +10,39 @@ set(oqc_backend_dir "${OQC_BUILD_DIR}/backend")
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-# Locate Python & PyBind11
-# The optional component is only used for the C++ test suite (to spin up their its interpreter),
+# nanobind suggests including these lines to configure CMake to perform an optimized release build
+# by default unless another build type is specified. Without this addition, binding code may run
+# slowly and produce large binaries.
+# See https://nanobind.readthedocs.io/en/latest/building.html#preliminaries
+if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+    set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+endif()
+
+# Locate Python
+# The optional component is only used for the C++ test suite (to spin up its own interpreter),
 # and requires libpython.so to be available on the system.
 find_package(Python REQUIRED
     COMPONENTS Interpreter Development.Module
     OPTIONAL_COMPONENTS Development.Embed
```

Oh my bad, I may have inadvertently removed the `Development.SABIModule` from our CMake files. I think we're supposed to include it whenever `nanobind_add_module(oqc_python_module STABLE_ABI` appears in a CMake project.
```suggestion
    OPTIONAL_COMPONENTS Development.Embed Development.SABIModule
```
Are you able to update the other projects as well?

---

Yes, I have added it where appropriate. Done.